README.md

```diff
@@ -9,11 +9,11 @@ Our journey to transend at Hive
 
 ## How to run
 
-https://localhost/ - this is the lobby view, where most of the interactions, login and user dashboards will happen. its equivalent to https://localhost/index.html
+https://localhost:8443/ - this is the lobby view, where most of the interactions, login and user dashboards will happen. its equivalent to https://localhost:8443/index.html
 
 ### Game
```

Comment:
We do not use game.html at all. Our project implements SPA.
The file `game.html` should be removed completely. I think it is some legacy stuff. Therefore, we do not even need this Game part in our README.md